homeassistant/components/danfoss_air/binary_sensor.py

```diff
@@ -32,28 +36,13 @@ class DanfossAirBinarySensor(BinarySensorEntity):
     def __init__(self, data, name, sensor_type, device_class):
         """Initialize the Danfoss Air binary sensor."""
         self._data = data
-        self._name = name
-        self._state = None
+        self._attr_name = name
```

Comment:
This changes the behavior of this integration. Is this intentional?

Reply:
Can you explain how replacing the name property with `_attr_name` changes the behaviour?
I don't want to make the same mistake again...

Reply:
Sorry, my bad, you are right.

The name property isn't dynamic. I'll pick up another cup of joe ☕